teams/codebender.md

```diff
@@ -19,14 +19,17 @@ permalink: /teams/codebender/
 
 ## Workplan
 
-* [ ] Project set up
-* [ ] Domain understood
+* [x] Domain understood - Week 1, Thu
+* [x] Product demo - Week 2, Mon
+* [x] Project set up - Week 2, Tue
 * [ ] Feature planned
 
 ## Contributions
  
 #### @thngkaiyuan (Thng Kai Yuan)
-* ...
+### Technology expertise in Information Security
+ * Sharing on Cross-Site Request Forgeries ([Blog Post on CSRF](http://cs3281-2.blogspot.sg/2016/01/cross-site-request-forgeries-csrf.html))
+ * Write-up on my participation in Insomni'hack 2016: ([Blog Post on the Smartcat Challenge](http://cs3281-2.blogspot.sg/2016/01/te-insomnihack-2016-teaser-smartcat-web.html))
```

Comment:
May be a summary? e.g. attended 3 training sessions, gave a talk on XYZ, 
If in doubt, you can err on the side of too much details. It's easy for us to tell you when there's too much details. Much harder to say anything when details are missing altogether. :-)